test/importer/MusicXmlImporter.test.ts

```diff
@@ -94,4 +95,106 @@ describe('MusicXmlImporterTests', () => {
         expect(score.tracks[0].staves[0].bars[0].voices[0].beats[0].chord!.strings[4]).to.equal(3);
         expect(score.tracks[0].staves[0].bars[0].voices[0].beats[0].chord!.strings[5]).to.equal(-1);
     });
+    it('bend', async () => {
+        let score: Score = await MusicXmlImporterTestHelper.testReferenceFile(
+            'test-data/musicxml-testsuite/100a-Guitare-Bends.xml'
+        );
+        let note = score.tracks[0].staves[0].bars[0].voices[0].beats[0].notes[0]
+        //expect(note.bendType).to.equal(BendType.Bend);
```

Comment:
Is this intentionally commented or a mistake? Feels like these check should be active. 
From the bend type detection [here](https://github.com/CoderLine/alphaTab/blob/ee200f047d1cad2ed5a0f499500061c10fe74eb5/src/model/Note.ts#L797-L870) we might need some extensions to handle the right combinations. 

Would be great if we can add support for correct bendTypes in these combinations. This will improve the display and playback of bends. 

What do you think?

Reply:
Yeah this is intentional, the bend type detection is currently very tied to GP 4 points bends logic. I'll take a look and see what can be done to make that more generic and try to make the test pass with uncommented bend type checks, I'll keep you updated.

Reply:
Yes, the current bend detection is relying on some patterns mainly coming from Guitar Pro. But from your test data I think it should be possible to extend the current list with some new variants from MusicXML. As long we just extend the list with new validations we should be fine. If we need to modify existing ones, we might have to double check how to align the different formats. 

Just let me know if you face any troubles 😉

Reply:
Okay, from the first tests I've made I noticed it would also add bend detection for old gp versions (3-5). I guess it's a good thing but I'll have to update the tests accordingly. "As long we just extend the list with new validations" : what list are you referring to ?

Reply:
Jup, having the detection for all formats is a good thing. 

> As long we just extend the list with new validations

Validations might have been the wrong term. Rephrased: If we add new checks to detect new combinations thats good. If we would need to modify any existing detection to match the MusicXML specifics this would be bad and we would have to check how to handle this.

Reply:
oh ok I get it. However I modified the bend type detection not really to handle the MusicXML specifics but rather to be more generic and allow an arbitrary number of bend points. That's also why it happens to work for GP3-5 too and doesn't break anything for GP6+ as far as I know.